src/RazorSdk/SourceGenerators/RazorSourceGenerationOptions.cs

```diff
@@ -49,11 +49,14 @@ internal class RazorSourceGenerationOptions : IEquatable<RazorSourceGenerationOp
         public bool SupportLocalizedComponentNames { get; set; } = false;
 
         public bool Equals(RazorSourceGenerationOptions other)
+                   => SuppressRazorSourceGenerator == other.SuppressRazorSourceGenerator && EqualsIgnoringSupression(other);
+
+        public bool EqualsIgnoringSupression(RazorSourceGenerationOptions other)
```

Comment:
>EqualsIgnoringSupression [](http://example.com/codeflow?start=20&length=24)

Should this consider `WaitForDebugger` as well?

Reply:
It was no longer used, so have removed.